fhttp/http.go

```diff
@@ -216,6 +221,10 @@ func NewStdClient(o *HTTPOptions) Fetcher {
 	if o.NumConnections < 1 {
 		o.NumConnections = 1
 	}
+	// 0 timeout for stdclient doesn't mean 0 timeout... so just warn and leave it
```

Comment:
explain what the effects of timeout of 0, negative value

Reply:
tbh I have to look it up
initially my test didn’t fail when I commented out the code that was setting the timeout to the default like I did for the fast custom client 

and tried with -1 and that also passed locally, I assumed it meant infinite timeout but then it failed on circle, so I set it back to unitialized/0